container-build-checks.py

```diff
@@ -67,7 +67,7 @@ def verify_reference(image, result, value):
     if f"{repo}:{tag}" not in image.containerinfo["tags"]:
         tags = ", ".join(image.containerinfo["tags"])
         result.warn(f"The org.opensuse.reference label ({value}) does not refer to an existing tag ({tags})")
-    elif image.containerinfo["release"] not in tag:
+    elif image.containerinfo.get("release", None) not in tag:
```

Comment:
With this it won't show this warning in all local builds:

```suggestion
    elif "release" in image.containerinfo and image.containerinfo["release"] not in tag:
```

The warning below still shows up if the release info is missing, so it'll still block builds if it's missing in OBS for whatever reason.